transport/src/main/java/io/netty/channel/IoHandleEventLoop.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+
+import io.netty.util.concurrent.Future;
+
+/**
+ * {@link EventLoop} that allows to register / deregister {@link IoHandle} instances.
+ */
+public interface IoHandleEventLoop extends EventLoop, IoHandleEventLoopGroup {
```

Comment:
I wonder if the name is too long and we should just call it `IoEventLoop` and so follow the same pattern for others like:

- `IoEventLoopGroup`
- `SingleThreadIoEventLoopGroup`
- `MultiThreadIoEventLoopGroup`

This would make instancing these a bit less verbose

Reply:
Makes sense, so the `Io` part is the common theme between `IoHandle` and `IoEventLoop`. I'm fine either way, though.

Reply:
Exactly... let's do it